packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -312,7 +312,9 @@ async function deleteScratchOrg(
 
   try {
     let communityUrl = '';
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (process.env.CI) {
```

Comment:
Kinda the same thing but  in this file we already have a function called `isCi`.
```suggestion
    if (isCi()) {
```